hmda/src/main/scala/hmda/api/http/admin/InstitutionAdminHttpApi.scala

```diff
@@ -141,10 +152,16 @@ trait InstitutionAdminHttpApi extends HmdaTimeDirectives {
     }
 
   val institutionReadPath =
-    path("institutions" / Segment) { lei =>
-      val institutionPersistence =
-        sharding.entityRefFor(InstitutionPersistence.typeKey,
-                              s"${InstitutionPersistence.name}-$lei")
+    path("institutions" / Segment / "year" / Segment) { (lei, period) =>
```

Comment:
Can we not allow institutions to be added for future years that the platform does not yet support? So at the moment anything greater than 2019 should throw an error. We can use the `hmda.filing.current` property in `application.conf` in common